src/features/projects/screens/SingleProjectDetails.tsx

```diff
@@ -109,7 +103,7 @@ function SingleProjectDetails({}: Props): ReactElement {
                   ? `?embed=true&callback=${callbackUrl}`
                   : '?embed=true'
               }`
-            : ''
+            : `/profile`
```

Comment:
I don't think this is a good solution as well. Of course from "/profile/history" jumping to a project this PR will return to "/profile" by this change. But the back link on a project site normally should navigation back to the home page of planet-webapp (just "/") and not forcing someone to login to show a profile. Especially for people visiting the site not having any profile at all this is extremely irritating!